confidential/confidential_test.go

```diff
@@ -4,12 +4,13 @@ import (
 	"crypto/rand"
 	"encoding/hex"
 	"encoding/json"
-	"github.com/stretchr/testify/assert"
-	"github.com/vulpemventures/go-secp256k1-zkp"
 	"io/ioutil"
 	"math/big"
 	"strconv"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/vulpemventures/go-secp256k1-zkp"
```

Comment:
why you need to import this?

Reply:
they were already imported, I just reordered the imports so that we kee p builtin packages and "external" ones separated.

Anyway, assert is used to check that results match the expected values, while go-secp256k1-zkp is used to parse pedersen commitments from string vectors to *Commitment instances.